ppr-ui/tests/unit/MhrSubmittingParty.spec.ts

```diff
@@ -139,13 +139,13 @@ describe('mhr submitting party', () => {
     expect(submittingPartySection.findAll(ERROR_MSG).length).toBe(0)
 
     const phoneNum = submittingPartySection.find('#submitting-party-phone')
-    phoneNum.setValue('(778) 123-456')
+    phoneNum.setValue('(778) 123-4563')

```

Comment:
This doesn't exactly adhere to the rule you've defined above, `must contain numbers only` so i'm a little confused.

Reply:
Good point... I will change that test.  The test is for numbers so I should test for that message...